rasa_nlu/model.py

```diff
@@ -121,11 +122,31 @@ def __init__(self, config, component_builder=None, skip_validation=False):
         if not self.skip_validation:
             components.validate_requirements(config.pipeline)
 
-        # Transform the passed names of the pipeline components into classes
+        # build pipeline
+        self.pipeline = self._build_pipeline(config, component_builder)
+
+    @staticmethod
+    def _build_pipeline(config, component_builder):
+        # type: (RasaNLUConfig, ComponentBuilder) -> List
+        """Transform the passed names of the pipeline components into classes"""
+        pipeline = []
+
         for component_name in config.pipeline:
-            component = component_builder.create_component(
-                    component_name, config)
-            self.pipeline.append(component)
+            try:
+                component = component_builder.create_component(
+                        component_name, config)
+                pipeline.append(component)
+            except NoLanguageSupportingError as e:
+                # component don't support current language
+                if not config.component_force_language_support:
```

Comment:
Do we need to support this? E.g. what would be the use case to ignore a components limitations?

Additionally, I am not sure this is doing what the user expects, since the component that doesn't support the language will just be missing from the created pipeline

Reply:
I add this switch variable because at least unittest `tests/base/test_interpreter.py::test_interpreter[all_components]` will fail since this test use langauge=en by default while component proposal by #477 does not support this language. It will raise exception.